resources/js/assets/lang/pl.json

```
@@ -0,0 +1,80 @@
+{
+    "GUI": {
+        "Prices" : "Ceny",
+        "Find_a_ride" : "Znajdź przejazd",
+        "Rules" : "Zasady",
+        "Loading" : "Ładowanie",
+        "Admin_panel" : "Panel administratora",
+        "Close_menu" : "Zamknij menu",
+        "Open_menu" : "Otwórz menu",
+        "Search_city" : "Szukaj miasta",
+        "Search_country" : "Szukaj kraju",
```

```suggestion
        "Search_country" : "Szukaj kraju"
```

JSON standard does not allow trailing comma  - please fix it in other places.

---

fixed